packages/components/src/date-time/style.scss

```diff
@@ -7,6 +7,10 @@
 .components-datetime {
 	padding: $grid-unit-20;
 
+	.components-popover__content & {
```

Comment:
I recall having a conversation about keeping the padding behavior isolated to the context. But this feels a decent way to tune the behavior. Correct me if I'm wrong, Riad.

Reply:
Well ideally the component works the same in a popover or outside a popover. This is to say it shouldn't have padding IMO but I do remember the same discussion previously and you said that the calendar styles (third-party) kind of forces us to do have that padding somehow?

Reply:
Yes that was the case due to how the month pagination works. However right at this moment I can't reproduce. Let me try a thing.

Reply:
The issue we had back then does not appear to be an issue anymore, so I've pushed a different fix in https://github.com/WordPress/gutenberg/pull/20824/commits/c7302b7971f2dee277df4ec2732606e6deb5189c

And things look right:

<img width="351" alt="Screenshot 2020-03-12 at 12 02 13" src="https://user-images.githubusercontent.com/1204802/76515447-e2e8dc80-6459-11ea-945c-e4ec5b3e6fc1.png">

<img width="372" alt="Screenshot 2020-03-12 at 12 02 04" src="https://user-images.githubusercontent.com/1204802/76515459-e5e3cd00-6459-11ea-9c50-cd0bf80dd956.png">